### PR TITLE
Update version for the next release (v0.1.9)

### PR DESCRIPTION
**Pull request by bidoubiwa**
Update version to v0.1.9 to apply CI changes on publishing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.9
+
+Ensure build of project before publishing to npm #127
```
Comment (alallema) [inline CHANGELOG.md:3]: This one #124 doesn't appear in the Changelog?

**Review by alallema — COMMENTED** [review]
:tada !

File: CHANGELOG.md (hunk 1 above)
Comment (bidoubiwa) [reply CHANGELOG.md:3]: Its in the previous release

Comment (alallema) [reply CHANGELOG.md:3]: Oh. yes I didn't see it

**Review by alallema — APPROVED** [review]
LGTM! 🍗

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**